projects/demo/src/modules/experimental/badge/examples/1/index.html

```diff
@@ -0,0 +1,63 @@
+<tui-badge
+    appearance="primary"
+    value="Primary"
+    class="tui-space_horizontal-2 tui-space_vertical-2"
+></tui-badge>
+
+<tui-badge
+    appearance="accent"
+    value="Accent"
+    class="tui-space_horizontal-2 tui-space_vertical-2"
+></tui-badge>
+
+<tui-badge
+    appearance="dark-text"
+    value="Dark text"
+    class="tui-space_horizontal-2 tui-space_vertical-2"
```

Comment:
too many information in this example 🤔 

Should we move all `tui-space_horizontal-2` and `tui-space_vertical-2` to `Less`-file?

Something like:

```css
tui-badge {
    margin: 0.5rem;
}
```